src/components/Header.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable linebreak-style */
```

Comment:
- [ ] As requested in the previous review kindly remove this eslint disable line on all components. To fix it you can simply try running `npx eslint . --fix` if on a windows system, that usually resolves it on my end.

Reply:
its not working i tried it

Reply:
Well Addressed!!!